src/app/public/modules/vertical-tabset/vertical-tab.component.ts

```diff
@@ -169,7 +218,20 @@ export class SkyVerticalTabComponent implements OnInit, OnDestroy {
     }
   }
 
+  @HostListener('window:resize', ['$event'])
```

Comment:
Do we need the `event`? Can we just remove it?

Reply:
We don't need it, good catch. Removed.